crates/sui-indexer/migrations_v2/2023-10-06-204335_tx_recipients/up.sql

```diff
@@ -0,0 +1,8 @@
+-- Your SQL goes here
+CREATE TABLE tx_recipients (
+    tx_sequence_number          BIGINT       NOT NULL,
+    checkpoint_sequence_number  BIGINT       NOT NULL,
```

Comment:
All of these tables have checkpoint sequence numbers in them, is there an intended use case for this column, other than as a watermark?

Reply:
discussed with @gegaowp offline. My initial thinking is the checkpoint_sequence_number here will help with CSR. But now i'm not very confident planner will use this column wisely. Or if joining it with `transactions` table is easier. I'm happy to remove them now, with a small caveat that we may potentially need to add them back eventually if they are useful.